plugin/src/main/java/net/dzikoysk/funnyguilds/guild/top/GuildComparator.java

```diff
@@ -0,0 +1,37 @@
+package net.dzikoysk.funnyguilds.guild.top;
+
+import java.util.Comparator;
+import net.dzikoysk.funnyguilds.guild.GuildRank;
+
+public final class GuildComparator {
+
+    public static final Comparator<GuildRank> AVG_POINTS_COMPARATOR = new AveragePoints();
```

Comment:
This probably should be reversed by default instead of reversing passed comparator into recalculation functions (see `GuildRecalculation#apply` for example).
```suggestion
    public static final Comparator<GuildRank> AVG_POINTS_COMPARATOR = new AveragePoints().reversed();
```